x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/timeline_entry.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import moment from 'moment';
+
+import { EuiAvatar, EuiComment, EuiText } from '@elastic/eui';
+import { Immutable, EndpointAction } from '../../../../../../../common/endpoint/types';
+
+export const TimelineEntry = memo(
+  ({ endpointAction }: { endpointAction: Immutable<EndpointAction> }) => {
+    const isIsolated = endpointAction?.data.command === 'isolate';
+
+    // do this better when we can distinguish between endpoint events vs user events
+    const iconType = endpointAction.user_id === 'sys' ? 'dot' : isIsolated ? 'lock' : 'lockOpen';
+    const commentType = endpointAction.user_id === 'sys' ? 'update' : 'regular';
+    const timelineIcon = (
+      <EuiAvatar
+        name="Timeline Icon"
+        size={endpointAction.user_id === 'sys' ? 's' : 'm'}
+        color="#f5f7fa"
+        iconColor="subdued"
+        iconType={iconType}
+      />
+    );
+    const event = `${isIsolated ? 'isolated' : 'unisolated'} host`;
+    const hasComment = !!endpointAction.data.comment;
+
+    return (
+      <EuiComment
+        type={commentType}
+        username={endpointAction.user_id}
+        timestamp={moment(endpointAction['@timestamp']).fromNow().toString()}
```

Comment:
We should be using our own relative date component - 

https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/plugins/security_solution/public/common/components/endpoint/formatted_date_time.tsx#L11-L25

Looks like `timestamp` prop accepts a ReactNode, so there should be no problem defining a component here

Reply:
done e38e202e81f8dd2796bebe9069175d31052c52f2